src/it/scala/uk/gov/hmrc/hmrcfrontend/views/components/HmrcAddToAListIntegrationSpec.scala

```diff
@@ -11,7 +11,7 @@ import uk.gov.hmrc.support.TemplateIntegrationBaseSpec
 
 import scala.util.Try
 
-object HmrcAddToAListSpec
```

Comment:
renamed these to match the prevailing convention, and because Intellij complained of name collisions